### DISCUSSION TITLE
Welcome to ao-starter-kit Discussions!

### DISCUSSION THREAD

**Discussion by micovi** [Autonomous-Finance/ao-starter-kit#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by 0xSSDD** [comment]
Hey guys. hope you are all doing well.  I am 0xSSDD, more info about me can be found on my github page :)

Absolutely love what you have built here, it is a great and very simple way to get into using AO, and related tech.

I faced a small issue with setting it up,  and I thought my fix could help others as well, here is the PR: https://github.com/Autonomous-Finance/ao-starter-kit/pull/20

Should I also open a separate issue for this?
